README.rst

```diff
@@ -257,17 +257,23 @@ change the behavior of what is returned from a decorated method. The default
 mapper can combine multiple records into a single result if there is an
 ``id`` field present in each record. Mappers available:
 
-* ``RecordCombiningMapper`` (default) - returns a list of results, with multiple records with the same ``id`` value
-  being combined into a single result. By default the column ``id`` is used but an array of column names can be used
-  to create a unique key lookup for combining records. The id value can be set as one or more columns in the mapper.
-  An optional ``record_mapper`` value may be passed to the constructor to change
-  how records are mapped to result. By default the ``record_mapper`` used is ``DbMapResult``.
+* ``RecordCombiningMapper`` (default) - Returns a list of results where multiple records that can be combined with the
+  same unique identifer. An optional ``record_mapper`` value may be passed to the constructor to change
+  how records are mapped to result. By default the ``record_mapper`` used is ``DbMapResult``. The base identifier
+  is the column ``id`` but an array of columns can be used to create a unique key lookup for combining records.
+
+.. note::
+    The ``_key_columns`` field of the ``DbMapResultModel`` in array containing only the ``id`` but can
```

Comment:
`in array containing only the ``id`` but can` maybe this should be `is an array`? I'm not sure what you're trying to say here.